src/languages/en-US.ts

```diff
@@ -3484,6 +3522,38 @@ export default class extends Language {
 		COMMAND_STAR_TOPRECEIVERS: 'Top Star Receivers',
 		COMMAND_STAR_TOPRECEIVERS_DESCRIPTION: (medal, id, stars) => `${medal}: <@${id}> (${stars} ${stars === 1 ? 'star' : 'stars'})`,
 
+		/**
+		 * ##################
+		 * SUGGESTIONS COMMANDS
+		 */
+		COMMAND_SUGGEST_NOSETUP: username => `I'm sorry ${username}, but a suggestions channel hasn't been setup.`,
+		COMMAND_SUGGEST_NOSETUP_ASK: username => `I'm sorry ${username}, but a suggestions channel hasn't been setup. Would you like to set-up a channel now?`,
+		COMMAND_SUGGEST_NOSETUP_ABORT: 'Alright then. Aborted creating a new suggestion.',
+		COMMAND_SUGGEST_CHANNEL_PROMPT: 'Please mention the channel you want to set as the suggestions channel. You have 30 seconds.',
+		COMMAND_SUGGEST_CHANNEL_INVALID: 'Didn\'t receive a valid channel mention. Aborting...',
+		COMMAND_SUGGEST_TITLE: id => `Suggestion #${id}`,
+		COMMAND_SUGGEST_SUCCESS: channel => `Thank you for your suggestion! It has been successfully posted to the ${channel} channel`,
+
+		COMMAND_RESOLVESUGGESTION_INVALID_ID: 'That\'s not a valid suggestion ID!',
+		COMMAND_RESOLVESUGGESTION_MESSAGE_NOT_FOUND: 'Couldn\'t find the suggestion\'s message. To prevent clutter, I have removed it from the database.',
+		COMMAND_RESOLVESUGGESTION_ID_NOT_FOUND: 'Couldn\'t find a suggestion with that ID',
+
+		COMMAND_RESOLVESUGGESTION_DEFAULT_COMMENT: 'No comment was provided.',
+		COMMAND_RESOLVESUGGESTION_AUTHOR_ADMIN: 'An administrator',
+		COMMAND_RESOLVESUGGESTION_AUTHOR_MODERATOR: 'A moderator',
+		COMMAND_RESOLVESUGGESTION_ACTIONS: {
+			ACCEPT: author => `${author} accepted this suggestion:`,
+			CONSIDER: author => `${author} considered this suggestion:`,
+			DENY: author => `${author} denied this suggestion:`
+		},
+		COMMAND_RESOLVESUGGESTION_ACTIONS_DMS: {
+			ACCEPT: (author, guild) => `${author} accepted this suggestion in ${guild}:`,
+			CONSIDER: (author, guild) => `${author} considered this suggestion in ${guild}:`,
+			DENY: (author, guild) => `${author} denied this suggestion in ${guild}:`
+		},
+		COMMAND_RESOLVESUGGESTION_DM_FAIL: 'I couldn\'t DM the user. Are their DMs closed?',
```

Comment:
Using emojis would be a good idea.
```suggestion
		COMMAND_RESOLVESUGGESTION_DM_FAIL: `${REDCROSS} I wasn't able to send the author a Direct Message. Are their DMs closed?`,
```